### DISCUSSION TITLE
reload hosts without using the /reload route

### DISCUSSION THREAD

**Discussion by AvivMark** [prometheus/prometheus#12069] [Q&A]
### Proposal

can you reload hosts without using the /reload route?

**Comment by roidelapluie** [comment]
Sure, instead of using the `/reload` route, I would suggest using file-based service discovery using file_sd. You can use a file and configure Prometheus to scrape targets from it periodically. This is a good option if your list of targets changes frequently or if you want to update the targets list without triggering a reload. Here is the documentation that covers all the details: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#file_sd_config
